WooCommerce/WooCommerceUITests/Tests/ProductsTests.swift

```diff
@@ -15,7 +15,7 @@ final class ProductsTests: XCTestCase {
     func testProductsScreenLoad() throws {
         let products = try GetMocks.readProductsData()
 
-        _ = try TabNavComponent()
+        try TabNavComponent()
```

Comment:
Just learned that by adding `@discardableResult` to the functions being called here, we no longer need to assign it to an `_`